modules/bootcamp/detect_landing_pad.py

```
@@ -86,33 +86,39 @@ def run(self, image: np.ndarray) -> "tuple[list[bounding_box.BoundingBox], np.nd
         # * conf
         # * device
         # * verbose
-        predictions = ...
+        predictions = self.__model.predict(image, conf=0.7, device=self.__DEVICE, verbose=True)
 
         # Get the Result object
-        prediction = ...
+        prediction = predictions[0]
 
         # Plot the annotated image from the Result object
         # Include the confidence value
-        image_annotated = ...
+        image_annotated = prediction.plot(conf=True)
 
         # Get the xyxy boxes list from the Boxes object in the Result object
-        boxes_xyxy = ...
+        boxes_xyxy = prediction.boxes.xyxy
 
-        # Detach the xyxy boxes to make a copy,
+        # Detach the xyxy boxes to make a copy,s
         # move the copy into CPU space,
         # and convert to a numpy array
-        boxes_cpu = ...
+        boxes_cpu = boxes_xyxy.cpu().numpy()
 
         # Loop over the boxes list and create a list of bounding boxes
         bounding_boxes = []
         # Hint: .shape gets the dimensions of the numpy array
         # for i in range(0, ...):
+        # print(boxes_cpu.shape[0])
+        for i in range(0, boxes_cpu.shape[0]):
             # Create BoundingBox object and append to list
             # result, box = ...
+            result, box = bounding_box.BoundingBox.create(boxes_cpu[i])
+            print(box)
+            if result:
+                bounding_boxes.append(box)
```

instead of skipping a failed box creation, we actually want to return nothing for the whole image